src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -50,18 +64,80 @@ void createStation() {
         assertThat(stationNames).containsAnyOf("강남역");
     }
 
-    /**
-     * Given 2개의 지하철역을 생성하고
-     * When 지하철역 목록을 조회하면
-     * Then 2개의 지하철역을 응답 받는다
-     */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("등록된 모든 지하철역을 조회한다.")
+    @Test
+    void findAllStations() {
+        // Given 2개의 지하철역을 생성하고
+        StationApi.create(ENDPOINT, new JsonBodyParams("name", "강남역"));
+        StationApi.create(ENDPOINT, new JsonBodyParams("name", "서울대입구역"));
 
-    /**
-     * Given 지하철역을 생성하고
-     * When 그 지하철역을 삭제하면
-     * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
-     */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+        // When 지하철역 목록을 조회하면
+        List<String> stationNames =
+                RestAssured.given().log().all()
+                        .when().get("/stations")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+
+        // Then 2개의 지하철역을 응답받는다.
+        assertThat(stationNames).hasSize(2);
+        assertThat(stationNames).containsExactly("강남역", "서울대입구역");
+    }
+
+    @DisplayName("등록된 지하철을 삭제하면 삭제된 역은 조회할 수 없다.")
+    @Test
+    void deleteAllStations() {
+        // Given 지하철역을 생성하고
+        ExtractableResponse<Response> 강남역_생성_응답 = StationApi.create(ENDPOINT, new JsonBodyParams("name", "강남역"));
+        ExtractableResponse<Response> 서울대입구역_생성_응답 = StationApi.create(ENDPOINT, new JsonBodyParams("name", "서울대입구역"));
```

Comment:
지하철역 생성시 endpoint 와 Input 은 동일한데요
다음과 같은 구조로 개선해 보면 어떨까요? 😄 

```suggestion
        ExtractableResponse<Response> 강남역_생성_응답 = StationApi.create("강남역");
        ExtractableResponse<Response> 서울대입구역_생성_응답 = StationApi.create("서울대입구역");
```